css/index.css

```diff
@@ -40,6 +40,11 @@ a:hover {
   text-decoration: underline;
 }
 
+.icon {
+  width:1.4em;
```

Comment:
there should be a space between the colon and the value, to conform to the rest of the curriculum